addons/resource.language.en_gb/resources/strings.po

```diff
@@ -18153,3 +18153,23 @@ msgstr ""
 msgctxt "#38025"
 msgid "Choose information provider"
 msgstr ""
+
+#: system/settings/rbp.xml
+msgctxt "#38026"
+msgid "Decode the stereo stream from 3D files"
```

Comment:
@popcornmix, @jjd-uk any comment on my suggestion above for the settings label? What do you guys think?